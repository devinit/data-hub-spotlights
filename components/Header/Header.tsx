import React from 'react';

const Header = () => {
  return (
    <header role="banner" className="header">
      <div className="row">
        <a href="/" className="branding">Development Initiatives</a>
      </div>
    </header>
  );
};

export default Header;
