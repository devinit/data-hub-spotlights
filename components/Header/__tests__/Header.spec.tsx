/**
 * @jest-environment jsdom
 */
import '@testing-library/jest-dom/extend-expect';
import React from 'react';
import * as TestRenderer from 'react-test-renderer';
import Header from '../Header';

describe('Header', () => {
  test('renders correctly', () => {
    const renderer = TestRenderer.create(
      <Header/>
    ).toJSON();

    expect(renderer).toMatchSnapshot();
  });
});
