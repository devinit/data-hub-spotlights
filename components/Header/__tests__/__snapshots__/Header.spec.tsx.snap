// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<header
  className="header"
  role="banner"
>
  <div
    className="row"
  >
    <a
      className="branding"
      href="/"
    >
      Development Initiatives
    </a>
  </div>
</header>
`;
